src/Faker/Provider/de_DE/Address.php

```diff
@@ -16,7 +16,7 @@ class Address extends \Faker\Provider\Address
r', 'Holzminden', 'Hohenstein-Ernstthal', 'Rostock', 'Heinsberg', 'Stralsund', 'Hünfeld', 'Husum', 'Havelberg', 'Wismar', 'Höxter', 'Hoyerswerda', 'Herzberg', 'Ilmenau', 'Illertissen', 'Ingolstadt', 'Iserlohn', 'Jena', 'Jüterbog', 'Jessen', 'Jülich', 'Karlsruhe', 'Kronach', 'Kelheim', 'Kehl', 'Kemnath', 'Bad Kissingen', 'Bad Kreuznach', 'Kaiserslautern', 'Kleve', 'Klötze', 'Kamenz', 'Konstanz', 'Koblenz', 'Kötzting', 'Kassel', 'Kitzingen', 'Kulmbach', 'Kusel', 'Königs Wusterhausen', 'Kyritz', 'Leipzig', 'Ludwigsburg', 'Lobenstein', 'Lübz', 'Luckau', 'Lemgo', 'Lüneburg', 'Lüdinghausen', 'Bad Liebenwerda', 'Lichtenfels', 'Lübben', 'Lörrach', 'Lippstadt', 'Bad Langensalza', 'Lüdenscheid', 'Luckenwalde', 'Ludwigslust', 'München', 'Marienberg', 'Mainburg', 'Mallersdorf', 'Marktheidenfeld', 'Miesbach', 'Malchin', 'Magdeburg', 'Mettmann', 'Melsungen', 'Meißen', 'Melle', 'Meppen', 'Merseburg', 'Mellrichstadt', 'Bad Mergentheim', 'Meiningen', 'Mühlhausen', 'Miltenberg', 'Moers', 'Monschau', 'Mühldorf am Inn', 'Mittweida','Mayen', 'Nabburg', 'Naila', 'Nauen', 'Neubrandenburg', 'Nordhausen', 'Neuss', 'Neunburg vorm Wald', 'Neustadt an der Waldnaab', 'Northeim', 'Norden', 'Nördlingen', 'Neuruppin', 'Neustadt am Rübenberge', 'Nürtingen', 'Neu-Ulm', 'Niesky', 'Neustrelitz', 'Osterburg', 'Ochsenfurt', 'Olpe', 'Osterode am Harz', 'Oranienburg', 'Oberviechtach', 'Oschatz', 'Potsdam', 'Passau', 'Pfaffenhofen an der Ilm', 'Parsberg', 'Paderborn', 'Parchim', 'Peine', 'Pegnitz', 'Perleberg', 'Pinneberg', 'Pritzwalk', 'Plauen', 'Pößneck', 'Pirmasens', 'Pasewalk', 'Prenzlau', 'Querfurt', 'Quedlinburg', 'Regensburg', 'Rastatt', 'Ribnitz-Damgarten', 'Recklinghausen', 'Regen', 'Rehau', 'Roth', 'Riesa', 'Rochlitz', 'Rathenow', 'Rosenheim', 'Roding', 'Rockenhausen', 'Rothenburg ob der Tauber', 'Reutlingen', 'Rudolstadt', 'Ravensburg', 'Rottweil', 'Stuttgart', 'Schwandorf', 'Säckingen', 'Stadtsteinach', 'Saarbrücken', 'Strasburg', 'Schleiz', 'Stade', 'Sondershausen', 'Stendal', 'Sebnitz', 'Seelow', 'Scheinfeld', 'Senftenberg', 'Staßfurt', 'Sangerhausen', 'Schwäbisch Hall', 'Suhl', 'Siegen', 'Sigmaringen', 'Saulgau', 'Schmölln', 'Saarlouis', 'Schlüchtern', 'Bad Salzungen', 'Schwabmünchen', 'Schwerin', 'Soest', 'Schrobenhausen', 'Schongau', 'Soltau', 'Sömmerda', 'Sonneberg', 'Spremberg', 'Strausberg', 'Stadtroda', 'Steinfurt', 'Starnberg', 'Sternberg', 'Stade', 'Staffelstein', 'Stollberg', 'Sulzbach-Rosenberg', 'Schweinfurt', 'Schwarzenberg', 'Tecklenburg', 'Teterow', 'Torgau', 'Tirschenreuth', 'Tuttlingen', 'Tübingen', 'Uelzen', 'Ueckermünde', 'Uffenheim', 'Vechta', 'Vilsbiburg', 'Viersen', 'Viechtach', 'Vohenstrauß', 'Warendorf', 'Wittenberg', 'Worbis', 'Wiedenbrück', 'Werdau', 'Weimar', 'Wertingen', 'Wesel', 'Wolfenbüttel', 'Witzenhausen', 'Wittstock', 'Wolgast', 'Wolmirstedt', 'Wolfach', 'Wolfratshausen', 'Wernigerode', 'Waren', 'Weißenfels', 'Weißwasser', 'Wittmund', 'Waldmünchen', 'Wunsiedel', 'Wurzen', 'Wetzlar', 'Wanzleben', 'Zerbst', 'Zschopau', 'Zeulenroda', 'Zossen'
```

Comment:
Since we're in that file, could you split the line by capital letter, to make diffs more readable?